content/docs/develop/reference/bridge/version_check/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Daily Version Check
+description: Enable/Disable daily version check feature
+weight: 21
+keywords: [bridge]
+---
+
+The feature of a daily version check informs a Keptn user in the Keptn Bridge about:
+
+-	new Keptn versions;
+-	security patches for the installed Keptn deployment;
```

Comment:
Is the Keptn deployment the control plane? If so, can we please rename it?

Reply:
It is the control plane - changed the naming.